FBSDKCoreKit/FBSDKCoreKit/FBSDKAppEvents.m

```diff
@@ -523,8 +523,10 @@ + (void)updateUserProperties:(NSDictionary *)properties handler:(FBSDKGraphReque
     }
     return;
   }
+
+  NSString *appID = [FBSDKAppEvents loggingOverrideAppID] ?: [FBSDKSettings appID];
   NSDictionary *params = @{ @"data" : dataJSONString };
-  FBSDKGraphRequest *request = [[FBSDKGraphRequest alloc] initWithGraphPath:[NSString stringWithFormat:@"%@/user_properties", [FBSDKSettings appID]]
+  FBSDKGraphRequest *request = [[FBSDKGraphRequest alloc] initWithGraphPath:[NSString stringWithFormat:@"%@/user_properties", appID]
```

Comment:
I do agree to the idea, reusability is always a good option, but since the first method `updateUserProperties` is a class method, I cannot use the `[self appID]` instance method to retrieve the correct appID without accessing the FBSDKAppEvent singleton from the class method, which I don't really consider as an healthy option, I don't if there is any logical reason behind making `appID` an instance method.

`- (NSString *)appID
{
  return [FBSDKAppEvents loggingOverrideAppID] ?: [FBSDKSettings appID];
}`

The better option would be to switch the `appID` method from instance method to class method, since it really does not need to be an instance method, `[FBSDKAppEvents loggingOverrideAppID]` and `[FBSDKSettings appID]` being class methods as well.

I didn't want to make too big of a change to the file for the pull request, I can proceed and make appID a class method and update this pull request if you're fine with that change.

Thank you.